programs/drift/src/math/amm_spread.rs

```diff
@@ -228,6 +225,28 @@ pub fn calculate_spread_leverage_scale(
     Ok(effective_leverage_capped)
 }
 
+pub fn calculate_spread_revenue_retreat_amount(
+    base_spread: u32,
+    max_spread: u64,
+    net_revenue_since_last_funding: i64,
+) -> DriftResult<u64> {
+    // on-the-hour revenue scale
+    let revenue_retreat_amount =
+        if net_revenue_since_last_funding < DEFAULT_REVENUE_SINCE_LAST_FUNDING_SPREAD_RETREAT {
```

Comment:
$25 seems somewhat arbitrary tbh but not sure how to improve it

Reply:
maybe could be a function of the quote value of the liquidity offered?

if liquidity is ~$10k on bids/asks. hourly revenue retreat could be in blocks of $10k/X?